src/screens/Chatlist.svelte

```diff
@@ -102,7 +102,7 @@
 		</div>
 
 		<h1>Livechat</h1>
-		This is the public livechat. Messages in this chat don't save.
+		This is livechat. Post history isn't saved here.

```

Comment:
```suggestion
		Post history isn't saved here.

```

Saying livechat here is a bit redundant